### PR TITLE
Fixed some issues, removed all the unused code and removed all the loggin until developing a proper logging method

### PR DESCRIPTION
**Pull request by poltow**

### CODE REVIEW

File: MainSelectorComponent.py [hunk 1]
```diff
@@ -267,106 +242,86 @@ def update(self):
 			#	self.log_message(line.strip())
 		
 	def _setup_usermode(self, mode):
```
Comment (hdavid) [inline MainSelectorComponent.py]: maybe rename as setup_mode as it does not only setup user modes anymore

File: MainSelectorComponent.py [hunk 2]
```diff
@@ -267,106 +242,86 @@ def update(self):
...
-		if as_active:
+		if as_active:#Set up stop clip buttons and stop all clips button
 			if self._session._stop_clip_buttons != None:
 				for button in self._session._stop_clip_buttons:
 					button.set_enabled(as_active)
 				#	button.set_on_off_values("Session.StopClip", "DefaultButton.Disabled")
 				self._session.set_stop_track_clip_buttons(self._session._stop_clip_buttons)
-
+				
+				#last side button is stop all clip buttons ???
```
Comment (hdavid) [inline MainSelectorComponent.py]: yes last scene button is a stop all, if last row of the matrix is acting as stop buttons

**Comment by hdavid** [comment]
looks good ! can you fix the merge conflicts and i'll merge this one :)

**Comment by thomasf** [comment]
for the sake of readability you should probably not mix all-files-indent-changes in the same commits as other changes. It's much more handy to keep those kind of changes apart in separate commits.

**Pull request merged**